apps/condo/domains/meter/schema/PropertyMeter.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Generated by `createschema meter.PropertyMeter 'number:Text; numberOfTariffs:Integer; installationDate:DateTimeUtc; commissioningDate:DateTimeUtc; verificationDate:DateTimeUtc; nextVerificationDate:DateTimeUtc; controlReadingsDate:DateTimeUtc; sealingDate:DateTimeUtc; isAutomatic:Checkbox; organization:Relationship:Organization:CASCADE; property:Relationship:Property:CASCADE; resource:Relationship:MeterResource:CASCADE; meta:Json;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const get = require('lodash/get')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, find, getByCondition, getById } = require('@open-condo/keystone/schema')
+
+const { UNIQUE_ALREADY_EXISTS_ERROR } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/meter/access/PropertyMeter')
+const { AUTOMATIC_METER_NO_MASTER_APP, B2B_APP_NOT_CONNECTED, B2C_APP_NOT_AVAILABLE } = require('@condo/domains/meter/constants/errors')
+const { deleteReadingsOfDeletedMeter } = require('@condo/domains/meter/tasks')
+const { PropertyMeter: PropertyMeterAPI } = require('@condo/domains/meter/utils/serverSchema/index')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+
+const { resource, numberOfTariffs, installationDate, commissioningDate, verificationDate, nextVerificationDate, controlReadingsDate, sealingDate, isAutomatic, b2bApp } = require('./fields')
+
+
+const PropertyMeter = new GQLListSchema('PropertyMeter', {
+    schemaDoc: 'Resource meter installed on the entire apartment building',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+        numberOfTariffs,
+        installationDate,
+        commissioningDate,
+        verificationDate,
+        nextVerificationDate,
+        controlReadingsDate,
+        sealingDate,
+        isAutomatic,
+        resource,
+        b2bApp,
+
+        number: {
+            schemaDoc: 'Number of resource meter, such as "А03 9908"',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: async ({ context, operation, existingItem, resolvedData, fieldPath, addFieldValidationError }) => {
```

Comment:
Why you need this validate input hook, when you have a database constraint?

Reply:
You're right, it is not needed)) apparently this is some kind of forgotten code. I got it from schema `Meter`